apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+    response.ComponentFraud = 'Yes';
+  }
+  if (data['types-of-exploitation-prostitution'] === 'true') {
+    response.ComponentProstitution = 'Yes';
+  }
+  if (data['types-of-exploitation-prostitution'] === 'true') {
+    response.ComponentProstitution = 'Yes';
+  }
+  if (data['types-of-exploitation-child-exploitation'] === 'true') {
+    response.ComponentChildExploitation = 'Yes';
+  }
+  if (data['types-of-exploitation-taken-somewhere'] === 'true') {
+    response.ComponentSexuallyassaulted = 'Yes';
+  }
+  if (data['types-of-exploitation-forced-to-commit-crime'] === 'true') {
+    response.ComponentForcedCrime = 'Yes';
+  }
+  if (data['types-of-exploitation-organs-removed'] === 'true') {
+    response.ComponentOrgansRemoved = 'Yes';
+  }
+  if (data['types-of-exploitation-unpaid-household-work'] === 'true') {
+    response.ComponentDomesticServitude = 'Yes';
+  }
+  if (data['types-of-exploitation-other'] === 'true') {
+    response.ComponentDomesticServitude = data['other-exploitation-details'];
+  }
+
+  response.OtherVictims = data['any-other-pvs'];
+
+  response.ReportedCase = data['reported-to-police'] === 'yes' ? 'Yes' : 'No';
+  response['CaseContact.Police_force'] = data['reported-to-police-police-forces'];
+  response['CaseContact.Alternate_contact'] = data['fr-alternative-contact'];
+  response['CaseContact.Alternate_first_responder'] = data['fr-alternative-contact'];
+  response['CaseContact.Local_authority'] = data['local-authority-contacted-about-child-local-authority-name'];
+
+  response['CaseContactOrganisation.Police_force'] = data['reported-to-police-police-forces'];
+  response['CaseContactOrganisation.Local_authority']
+    = data['local-authority-contacted-about-child-local-authority-name'];
+
+  response['CaseContactEmail.Police_force'] = data['local-authority-contacted-about-child-local-authority-email'];
+  response['CaseContactEmail.Alternate_contact'] = data['fr-alternative-contact'];
+  response['CaseContactEmail.Alternate_first_responder'] = data['fr-alternative-contact'];
+  response['CaseContactEmail.Local_authority'] = data['local-authority-contacted-about-child-local-authority-email'];
+
+  // response['CaseContactCustom1.Police_force'];
+  // response['CaseContactFirstName.Alternate_contact'];
+  // response['CaseContactFirstName.Local_authority'];
+  // response['CaseContactAddress.Alternate_contact'];
+  // response['CaseContactTown.Alternate_contact'];
+  // response['CaseContactCounty.Alternate_contact'];
+  // response['CaseContactPostcode.Alternate_contact'];
+  // response['CaseContactCustom1.Alternate_contact'];
+
+  response['CaseContactPhone.Local_authority'] = data['fr-details-phone'];
+
+  response.SafeToEmail = data['pv-contact-details-email-check'] === 'true' ? 'Yes' : 'No';
+  response.SafeToPostal = data['pv-contact-details-post-check'] === 'true' ? 'Yes' : 'No';
+
+  response.CountryLabel = data['fr-location'];
+  // response.Country = data['fr-location'] //eg ENG;
```

Comment:
comment?

Reply:
yes because it's in the API but we don't know what to do with it yet